server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1593,11 +1593,8 @@ private ServiceLockData getManagerLock(final ServiceLockPath zManagerLoc)
     UUID zooLockUUID = UUID.randomUUID();
 
     ServiceDescriptors descriptors = new ServiceDescriptors();
-    for (ThriftService svc : new ThriftService[] {ThriftService.MANAGER,
-        ThriftService.COORDINATOR}) {
-      descriptors.addService(
-          new ServiceDescriptor(zooLockUUID, svc, managerClientAddress, this.getResourceGroup()));
-    }
+    descriptors.addService(new ServiceDescriptor(zooLockUUID, ThriftService.MANAGER,
+        managerClientAddress, this.getResourceGroup()));
```

Comment:
Only advertise the Manager service while starting up, clients can't connect until the Manager is fully up anyway.